src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSServiceHelper.cs

```diff
@@ -7,18 +7,88 @@ namespace OpenTelemetry.Instrumentation.AWS.Implementation;
 
 internal class AWSServiceHelper
 {
-    internal static IReadOnlyDictionary<string, string> ServiceParameterMap = new Dictionary<string, string>()
+    internal static IReadOnlyDictionary<string, List<string>> ServiceParameterMap = new Dictionary<string, List<string>>()
     {
-        { AWSServiceType.DynamoDbService, "TableName" },
-        { AWSServiceType.SQSService, "QueueUrl" },
+        { AWSServiceType.DynamoDbService, new List<string> { "TableName" } },
+        { AWSServiceType.SQSService, new List<string> { "QueueUrl" } },
+        { AWSServiceType.BedrockRuntimeService, new List<string> { "ModelId" } },
+        { AWSServiceType.BedrockAgentRuntimeService, new List<string> { "AgentId", "KnowledgeBaseId" } },
```

Comment:
I understand why you don't include bedrock and bedrockAgent in ServiceParameterMap. 
Can you please split `ServiceParameterMap` to `ServiceRequestParameterMap` and `ServiceResponseParameterMap`, then add bedrockAgent into ServiceRequestParameterMap, add bedrock into ServiceResponseParameterMap.